install_static_swift_modules/after/Pods/Headers/Private/CustomModuleMapPod/CustomModuleMapPod-library-iOS.modulemap

```diff
@@ -1 +1 @@
-/Users/segiddins/Development/Rainforest/CocoaPods/tmp/install_static_swift_modules/transformed/Pods/Target Support Files/CustomModuleMapPod-library-iOS/CustomModuleMapPod-library-iOS.modulemap
\ No newline at end of file
+/Users/derko/Development/cocoapods/CocoaPods/tmp/install_static_swift_modules/transformed/Pods/Target Support Files/CustomModuleMapPod-library-iOS/CustomModuleMapPod-library-iOS.modulemap
```

Comment:
could you undo these changes? its on our list of TODOs to make these use relative / mocked paths but for now would rather leave it at the current value